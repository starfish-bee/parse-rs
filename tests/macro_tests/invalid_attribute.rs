use parser::*;

#[derive(Operator, Debug, Clone, Copy)]
enum Op {
    #[assoc("right")]
    #[ident("#")]
    #[weird("")]
    Hash,
    #[assoc("right")]
    #[ident("^")]
    Caret,
    #[ident("?")]
    QMark,
}

fn main() {}
