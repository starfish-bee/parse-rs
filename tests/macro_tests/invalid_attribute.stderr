error: unrecognised attribute
 --> $DIR/invalid_attribute.rs:7:7
  |
7 |     #[weird("")]
  |       ^^^^^

error: cannot find attribute `weird` in this scope
 --> $DIR/invalid_attribute.rs:7:7
  |
7 |     #[weird("")]
  |       ^^^^^
